src/boundary/Obstacle.cpp

```diff
@@ -106,8 +90,8 @@ void Obstacle::init(size_t level) {
     m_obstacleLeft = new size_t[m_size_obstacleLeft];
     m_obstacleRight = new size_t[m_size_obstacleRight];
 
-    //m_size_obstacleInner = (m_strideX - 2) * (m_strideY - 2) * (m_strideZ - 2);
-    //m_obstacleInner = new size_t[m_size_obstacleInner];
+    // m_size_obstacleInner = (m_strideX - 2) * (m_strideY - 2) * (m_strideZ - 2);
+    // m_obstacleInner = new size_t[m_size_obstacleInner];
```

Comment:
is this code not longer needed -> delete

Reply:
5c97868f4303b2d83989a6c476cfcb68930c7c12